docs/logs/sending-container-logs.md

```diff
@@ -0,0 +1,53 @@
+title: Sending Container Logs
+description: Collect and ship docker logs 
+
+
+## Sending Container Logs
+
+[Logagent](https://sematext.com/logagent/) is a general purpose open-source log shipper. The [Logagent Docker image](https://hub.docker.com/_/logagent) is pre-configured for the [log collection on container platforms](https://sematext.com/blog/docker-container-monitoring-with-sematext/#toc-container-logs-0). It runs as a tiny container on every Docker host and collects logs for all cluster nodes and their containers. Logagent enriches container logs with container metadata such as Kubernetes, Docker Enterprise, and Docker Swarm.
+
+The deployment of Logagent is very flexible, all its [configurations options for logs](https://sematext.com/docs/logagent/installation-docker/#configuration-parameters). 
+
+Logagent, recognizes log formats from various applications / official images out of the box. The `patterns.yml file for [log parser patterns](https://sematext.com/docs/logagent/parser/) contains regular expressions to match containers name or image, regular expressions to extract fields, field names and optional JavaScript functions to transform the result. 
```

Comment:
Logagent, recognizes => no comma
The `patterns.yml file => missing ` ?
containers name or image => container names or images
the rest => hmmmm, the result of what?

Reply:
Forgot to close the code snippet. 
> The `patterns.yml (no closing backtick here, please add it) file for...